editor.planx.uk/src/@planx/components/DrawBoundary/Editor.tsx

```diff
@@ -53,25 +53,13 @@ function DrawBoundaryComponent(props: Props) {
               onChange={formik.handleChange}
             />
           </InputRow>
-          <InputGroup label="Boundary data field">
+          <InputGroup label="Data field">
             <InputRow>
               <Input
-                name="dataFieldBoundary"
-                placeholder=""
+                name="fn"
                 format="data"
-                value={formik.values.dataFieldBoundary}
-                onChange={formik.handleChange}
-              />
-            </InputRow>
-          </InputGroup>
-          <InputGroup label="Area data field (square metres)">
-            <InputRow>
-              <Input
-                name="dataFieldArea"
-                placeholder="property.boundary.area"
-                format="data"
-                value={formik.values.dataFieldArea}
-                onChange={formik.handleChange}
+                value={formik.values.fn}
+                disabled
```

Comment:
This file is a good summary / convergence point of main significant changes throughout: 
- Only support a single data field prop (rather than two data fields)
- Change the underlying prop name for the data field so it's picked up in graph styling, search facets by default
- Disable the data field input similar to PlanningConstraints & Pay as it's actually very important for integrations that it's called exactly this
- Change the actual data field _value_ to better align to schema

**Before:**
![Screenshot from 2024-12-17 17-59-28](https://github.com/user-attachments/assets/8672c7e4-8261-4491-af09-92414604aa6e)
![Screenshot from 2024-12-17 17-59-04](https://github.com/user-attachments/assets/a0f5b506-afc1-474d-83dd-b1d7a6c28ed0)

**After:** 
![Screenshot from 2024-12-17 17-58-28](https://github.com/user-attachments/assets/432d49c7-c64d-4096-94e4-4ce8136743b0)
![Screenshot from 2024-12-17 17-58-02](https://github.com/user-attachments/assets/037b3447-55ba-4468-9583-ffa6c5f963ae)